week2/homework/src/index.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+let fs = require('fs');
+
+let args = process.argv.slice(2);
+
+let command = args[0];
+let todoItem = args[1];
+
+
+
+if(command === 'add') {
+  add(todoItem);
+} else if (command === 'help'){
+  help();
```

Comment:
Try to run `node . help`; you will get an error

    ReferenceError: helpText is not defined

So you need to move the declaration of `helpText` to before the function call.